createdisk.sh

```diff
@@ -24,7 +24,7 @@ BASE_OS=${BASE_OS:-rhcos}
 OPENSHIFT_VERSION=$(${JQ} -r .clusterInfo.openshiftVersion $INSTALL_DIR/crc-bundle-info.json)
 BASE_DOMAIN=$(${JQ} -r .clusterInfo.baseDomain $INSTALL_DIR/crc-bundle-info.json)
 BUNDLE_TYPE=$(${JQ} -r .type $INSTALL_DIR/crc-bundle-info.json)
-destDirPrefix="crc"
+destDirPrefix=${destDirPrefix:-crc}
```

Comment:
BUNDLE_TYPE will be `okd` for an okd bundle, you can move the `if [[ ${OKD_VERSION} != "none" ]]` test here, check for BUNDLE_TYPE instead (possibly with a `case`) and make the okd/microshift code consistent. You can then have a `default` case which sets `BASE_OS` to `rhcos` and `destDirPrefix` to `crc`.